EnglishTranslation-master/SKILL.tsv

```diff
@@ -5744,7 +5744,7 @@ SKILL_20151224_005743	Increases attack equal to [Skill Level+1]x[Number of Weapo
 SKILL_20151224_005744	Penalty Reduction
 SKILL_20151224_005745	Reduces the penalty from incapable of combat.
 SKILL_20151224_005746	Equipment Durability Penalty: -#{CaptionRatio}#%{nl}Consumed Silver Penalty: -#{CaptionRatio2}#%{nl}Consumed Gem Penalty: -#{CaptionTime}#%
-SKILL_20151224_005747	Plant a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance. (Combo is only activated if the party leader is a Corsair.)
+SKILL_20151224_005747	Place a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance. (Combo is only activated if the party leader is a Corsair.)
```

Comment:
`And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance.`
maybe we should rework this to
`Attacking enemies within the jolly roger allows you to pillage items at a chance based on your Jolly Roger level.

Also, does every instance of "jolly roger" need to be capitalized in this line? if so, you should probably do that